models/user.js

```diff
@@ -87,8 +87,8 @@ UserSchema.static('registerToAccount', function(username, password, account, rol
 
 UserSchema.static('authenticate', function (username, password, callback) {
     // don't use cache() here, before a proper way to invalidate the cache when, e.g., the password is changed is
-    // implemented. See also: https://github.com/Gottox/mongoose-cache/issues/17
-    this.findOne({ username: username }).exec(function(err, user) {
+    // implemented. See also: https://github.com/Gottox/mongoose-cache/issues/17  
+	this.findOne({ username: { $regex: new RegExp("^" + username.toLowerCase(), "i") } }).exec(function(err, user) {
```

Comment:
Why is this changing in this PR?  Are you sure this branch is up to date

Reply:
This was from a PR before. And I had some problem with my fork.

Reply:
In any case it fixes another problem we have with case sensitive usernames.